src/TVar.h

```diff
@@ -67,6 +69,7 @@ class TVar
     bool saved;
 
 private:
+    int id = 0;
```

Comment:
:rotating_light: Caution, duplication:

Each `TVar` instance already has a `(int) mID` member (which was not being used!) and which could provide a unique positive integer identifier - it is inherited from the base `Tree` template defined in the `Tree.h` file - other Mudlet items get the number initialised via calls to their `setID(int)` methods which uniformly get the `int` required by calls to a `XxxxxUnit::getNewID()` method (where Xxxx is one of `Action/Alias/Key/Script/Timer/Trigger` as appropriate) - it seems to me that you should be using THAT value instead - likewise you don't need the `getId()`s and `setId(const int)` methods as there already are `getID() ` and `setID(int)` predefined for you.